lib/Cro/WebApp/Form.pm6

```diff
@@ -747,6 +761,9 @@ role Cro::WebApp::Form {
     method add-validation-error($message --> Nil) {
         $!validation-state.add-custom-error($message);
     }
+
+    #| Defines the prefix to use for translations using this form
+    method i18n-prefix(--> Str) { Str:U }
```

Comment:
I wonder if this shoulda been done as a trait on the form class for consistency with it being an attribute trait on the fields.

Reply:
I considered it as well, I can change it.

Reply:
:+1: